modules/javafx.controls/src/test/java/test/javafx/scene/control/TableViewTest.java

```diff
@@ -6329,4 +6329,54 @@ public void startEdit() {
 
         assertEquals(1, startEditCounter.get());
     }
+
+    @Test
+    void testReSetItemsWithSameItemShouldUpdateCellIndices() {
+        table.setFixedCellSize(24);
```

Comment:
fixed size is not essential to the test, right?  I've tested with the SCCE in the ticket, it has no effect.

it's probably ok to leave it here though.

Reply:
Ah yes, I used it initially to better predict the `ScrollBar`, but is indeed not necessary.